src/display/canvas.js

```diff
@@ -478,6 +472,7 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
     this.smaskCounter = 0;
     this.tempSMask = null;
     this.cachedCanvases = new CachedCanvases();
+    this.DOMcanvasFactory = new DOMcanvasFactory();
```

Comment:
I'm not sure that I really understand what you're asking here, but you should (obviously) be consistent within the file. I.e. the above ought to look like below (see also the code snippet in https://github.com/mozilla/pdf.js/pull/8002#discussion_r98327854).
```javascript
this.canvasFactory = new DOMCanvasFactory();
this.cachedCanvases = new CachedCanvases(this.canvasFactory);
```